content/en/community.md

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
Shouldn't this file be included in the `.gitignore` so that the latest version is pulled during build?

Reply:
Hum... not sure what to make of https://github.com/kubernetes-sigs/contributor-site/commit/69818fa3a79c39812044c8d095aa2eb2f3d4329f - I guess we'd need to do this in a similar fashion?

Reply:
Means that it should be excluded, see the combination of:

https://github.com/kubernetes-sigs/contributor-site/blob/69818fa3a79c39812044c8d095aa2eb2f3d4329f/content/en/docs/.gitignore + https://github.com/kubernetes-sigs/contributor-site/blob/69818fa3a79c39812044c8d095aa2eb2f3d4329f/external-sources/kubernetes/community#L17